src/components/Calculator.js

```diff
@@ -18,82 +18,89 @@ const  Claculator = () =>  {
 
    const { total, next, operation } = this.state;
```

Comment:
- [ ] When I navigate to the Calculator page the app crash and goes blank. Kindly look into that.

  app when I navigate to calculator route | error message in console
  :---:|:---:
  ![didioe](https://user-images.githubusercontent.com/54848363/206144135-933c2e68-5600-46ea-85c4-b7ab27c1b65c.jpg) | ![errir](https://user-images.githubusercontent.com/54848363/206144120-12fb9714-d72c-49e9-b172-3c547d88f0e0.jpg)

  You can fix this by removing every occurrence of `this.` in this component. You need to change this:
  ```
  const { total, next, operation } = this.state;
  ```
  to this:
  ```
  const { total, next, operation } = state;
  ```

  Also change every `this.eventHandler` to this `eventHandler`

Reply:
thanks for your suggestions and also for providing useful information for me. The issue was when I resolved the conflict, might happen that error, I will try to solve that now. And Again thanks for your time and effort.